### PR TITLE
fix: squad graphql query

### PR DESCRIPTION
**Pull request by kopancek**
## Changes

[More details on slack](https://dailydotdev.slack.com/archives/C06CUKX1P3N/p1715760110448429)

TLDR: our graphorm implementation cannot deal with children having timestamps without timezone. The `createdAt` field on `UserShortInfo` fragment was added to serve devcard requests. Easy fix was to remove the `createdAt` from the fragment and add the field directly to the devcard request. This is the easy fix.

Root cause fix would be to fix the graphorm implementation, but that's out of scope of public squad work.

### On those affected packages:
- [X] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

### Preview domain
https://fix-squad-graphql-query.preview.app.daily.dev

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
Thank you for spending the time on investigating this issue 🙏
